lib/ecto_mnesia/storage/migrator.ex

```diff
@@ -168,7 +170,9 @@ defmodule EctoMnesia.Storage.Migrator do
   # Helpers
   defp do_create_table(repo, table, type, attributes) do
     config = conf(repo)
+    attributes = if length(attributes) == 1, do: attributes ++ [:id], else: attributes
```

Comment:
there are one test from ecto with only one attribute, mnesia does not allow it, it need to have at least two attributes

Reply:
I think it would put us in a bad place if we do like that, people are not expecting that adapter would add any new fields to the table. Instead, we should raise an exception that this kind of migration is not supported and we can disable that particular test in Ecto itself.

Reply:
yes, it makes a lot of sense to not add new fields,

I found on ecto source, that it is the `Barebone` schema that has only one field:
https://github.com/elixir-ecto/ecto/blob/v2.2.0/integration_test/support/schemas.exs#L183

But I have no idea how to disable that particular test

Reply:
@prem-prakash I believe it's not the schema itself but it's usage in integration case for migrations (eg. [here](https://github.com/elixir-ecto/ecto/blob/8e493a5c56789dbaf974c6722b425a1d1c4cac6b/integration_test/support/migration.exs)). To workaround it you can open a PR to the Ecto itself and move that single schema to a separate test case with `:barebone_schemas` tag. So that we would be able to disable that case for ecto_mnesia in `text_helpers.ex`.

Reply:
uhn I got it, I will do that